App.tsx

```diff
@@ -1,21 +1,24 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
```

Comment:
please import what ever you want. dont import all because it takes more time to complie